src/stock_str.rs

```diff
@@ -424,6 +424,21 @@ Help keeping us to keep Delta Chat independent and make it more awesome in the f
 
 https://delta.chat/donate"))]
     DonationRequest = 193,
+
+    #[strum(props(fallback = "Outgoing Call"))]
```

Comment:
```suggestion
    #[strum(props(fallback = "Outgoing call"))]
```
Lowercase "c" looks more natural for me here

Reply:
Telegram does uppercase it, I am not native speaker, but we also have such upercasing for some titles in other strings 🤔 not sure what is the correct way heee

Reply:
Telegram just doesn't follow any rules here. It has "Clear History" and "Delete chat" in the same menu. AFAIU the common rule is to use upper case in names and "sentence case" in other text e.g. descriptions